docs/reference/api/docker_remote_api.md

```diff
@@ -154,7 +154,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 
 [Docker Remote API v1.23](docker_remote_api_v1.23.md) documentation
 
-* `GET /containers/json` returns the state of the container, one of `created`, `restarting`, `running`, `paused`, `exited` or `dead`.
+* `GET /containers/json` returns the state of the container, one of `created`, `restarting`, `running`, `removing`, `paused`, `exited` or `dead`.
```

Comment:
This shouldn't be there but a note for `Docker Remote API v1.25` should be added to say that a new state is there.